src/formula1_race_analysis/q1_session_analyzer.py

```diff
@@ -54,43 +58,58 @@ def read_file_content(filepath: Path) -> list[str]:
         raise MissedFileError(f"Error! The file path: {filepath} is not found or cannot be opened.") from error
 
 
-def parse_abbreviation_file(filepath: Path, ignore_errors: bool | None) -> list[Driver]:
+def create_driver_list(filepath: Path, ignore_errors: bool | None) -> list[Driver]:
     """
-    Parses a driver abbreviation file to create a driver database.
-    Raises InvalidFormatDataError when data in the file is not in the expected format.
+    Parses the driver abbreviation file and returns a list of Driver objects.
     """
+    drivers = []
     raw_data_from_abbreviation_file = read_file_content(filepath)
-    return format_data_from_abbreviation_file(raw_data_from_abbreviation_file, ignore_errors)
+    for line in raw_data_from_abbreviation_file:
+        try:
+            entry = AbbreviationEntry.model_validate(line)
+            drivers.append(Driver(**entry.model_dump()))
```

Comment:
drivers.append(Driver.from_pydantic_model(entry))

and create this from_pydantic_model method in Driver